src/tensor_ops/arith_broadcast_outer.rs

```diff
@@ -81,6 +83,98 @@ where
     binary_map_broadcast_rhs_first(lhs, rhs, div::f, div::dfdx, div::dfdy)
 }
 
+/// `lhs + &rhs`. `rhs` is broadcasted `N * M` times, where `N` is the first dimension and `N` is the second dimension of `lhs`.
+///
+/// E.g If Lhs has dimension `(5, 2, 3)`, then Rhs has to be dimension `(3,)`.
+///
+/// Examples:
+/// ```rust
+/// # use dfdx::prelude::*;
+/// let a = Tensor3D::new([[[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]]]);
```

Comment:
Could you add another array at the highest dimension? Just to cover that the same array is applied to all dimensions
e.g.
```rust
let a = Tensor3D::new([
        [[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]],
        [[-1.0, -2.0, -3.0], [-4.0, -5.0, -6.0]],
    ]);
```